Jellyfin.Server/Migrations/Routines/RemoveDownloadImagesInAdvance.cs

```diff
@@ -35,8 +35,14 @@ public void Perform()
             _logger.LogInformation("Removing 'RemoveDownloadImagesInAdvance' settings in all the libraries");
             foreach (var virtualFolder in virtualFolders)
             {
+                // Some virtual folders don't have a proper item id.
+                if (!Guid.TryParse(virtualFolder.ItemId, out var folderId))
+                {
+                    continue;
+                }
+
                 var libraryOptions = virtualFolder.LibraryOptions;
-                var collectionFolder = (CollectionFolder)_libraryManager.GetItemById(virtualFolder.ItemId);
+                var collectionFolder = (CollectionFolder)_libraryManager.GetItemById(folderId);
```

Comment:
A thought occurred to me. Playlists and Collections are also virtual folders, right? Do they have library options?

Reply:
@cvium The migration ran fine for me in a server with playlists and collections.

Reply:
I don't doubt that, I just wonder if it creates library options for non libraries 🤔

Reply:
@cvium I can do an stable-> unstable upgrade again.

I'll check once I arrive home and I'll report back

Reply:
@cvium Copied data from stable and re-ran the migration: only what's seen in the Dashboard > Libraries section is treated here. This includes the "Collections" library that gets created when you create a collection **but not the collections themselves**. All my playlists are also unaffected.